spark/src/main/res/values-fr/strings.xml

```diff
@@ -60,4 +60,10 @@
     <string name="spark_textfield_content_description">Obligatoire</string>
     <string name="spark_textfield_delete_a11y">Tout supprimer</string>
 
+    <string name="spark_text_link_paragraph_example"><annotation typography="body1" color="neutral">This is Adevinta </annotation><annotation typography="body1" color="main"><u><b>Privacy &amp; Policy</b></u></annotation><annotation typography="body1" color="neutral">also lots of extra information you maybe interested in or should I inform you about that extra information. you might be not knowing it</annotation> </string>
```

Comment:
I'm not sure why these are added to the public spark package. Shouldn't we only declare them in the catalog or screenshots for testing purposes?

Reply:
As it was used in the preview test, I will remove it